portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,43 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Essence Cain</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Hello!</h1>
+      <h2>I am Essence Cain</h2>
+      <h3>About Me</h3>
+      <p>I am a current Computer Science student at North Carolina A&T State University, where I have began to explore mobile development and UX/UI design.</p>
+      <p>I believe technology will have a strong impact in advancing our society and I want to play a key role by offering a different perspective from undereppresnted groups 
+          in STEM fields.</p>
+      <h3>Work Experience</h3>
+      <dl>
+          <img src ="/images/google_icon.jpeg"/>
+          <dt>Google</dt>
+          <dd>Student Training in Engineering Program Intern</dd>
+          <dd>May 2020 - Current</dd>
+      </dl>
+      <dl>
+          <dt>Google</dt>
+          <dd>Engineering Practicum Intern</dd>
+          <dd>May 2019 - August 2019</dd>
+      </dl>
+      <h3>Skills</h3>
+      <p>Java and C++</p>
```

Comment:
This looks like a list. Have you considered using the `<ul>` element? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul

Reply:
I changed it  from a p element to a dl